testsuite/features/step_definitions/common_steps.rb

```diff
@@ -696,8 +696,7 @@
 Then(/^I add proxy record into hosts file on "([^"]*)" if avahi is used$/) do |host|
   node = get_target(host)
   if node.full_hostname.include? 'tf.local'
-    output, _code = $proxy.run("ip address show dev eth0")
-    ip = output.split("\n")[2].split[1].split('/')[0]
+    ip = client_public_ip 'proxy'
```

Comment:
I personally prefer to call the method as you are doing here, it's more Ruby style.
But in the majority of our code we use parenthesis to call our methods, so, maybe better to use parenthesis?
Really just a matter of taste, but to try to keep some code style.

Reply:
I like this idea.

Reply:
maybe it even already exists.... who knows